factory/__init__.py

```diff
@@ -54,22 +55,10 @@
     stub_batch,
 )
 
-try:
-    from . import alchemy
-except ImportError:
-    pass
 try:
     from . import django
 except ImportError:
     pass
-try:
-    from . import mogo
-except ImportError:
-    pass
-try:
-    from . import mongoengine
-except ImportError:
-    pass
```

Comment:
These two submodules don’t have any library-specific imports, so the `try/except` was never needed.